packages/react/src/utils/floating-ui/useClick.ts

```diff
@@ -53,16 +54,7 @@ export function useClick(context: FloatingRootContext, props: UseClickProps = {}
   } = props;
 
   const pointerTypeRef = React.useRef<'mouse' | 'pen' | 'touch'>(undefined);
-  const frameRef = React.useRef(-1);
```

Comment:
Just fyi, `-1` is not recommended as an empty value for rAF: https://developer.mozilla.org/en-US/docs/Web/API/Window/requestAnimationFrame#return_value